Optik/Classes/ImageViewController.swift

```diff
@@ -30,7 +30,8 @@ internal final class ImageViewController: UIViewController {
         }
     }
     private(set) var imageView: UIImageView?
-
+    var likeButton: UIButton?
```

Comment:
The dismiss button is create in the setup method and the view controller doesn't  store any reference because the dismiss button doesn't need to be update. However the like button need to change the currentImage. That's the way I create a variable likeButton. I can initialize immediately the button  in the class
`var likeButton = UIButton( .custom)`

I don't know if I answer your question. If not, please, let me know then I can change the code properly